RationaleMCP/0031/functions.md

```diff
@@ -0,0 +1,95 @@
+# Built-in functions and operators
+
+## In Flat Modelica
+
+### No discussion needed
+
+* abs
+* sign
+* sqrt
+* div
+* mod
+* rem
+* ceil
+* floor
+* integer
+* sin
+* cos
+* tan
+* asin
+* acos
+* atan
+* atan2
+* sinh
+* cosh
+* tanh
+* exp
+* log
+* log10
+* initial
+* terminal
+
+### Probably should be in Flat Modelica as is
+
+* ndims
+* delay
+* spatialDistribution
+* homotopy
+* sample
+* size(A,i)
+* size(A)
```

Comment:
I suggest not seeing `size` as a function, as it makes more sense to think of as an expression operating on the type of the first argument rather than the value.